### PR TITLE
Allow specifying frequency lists in ZXCVBNValidator options

### PR DESCRIPTION
**Pull request by ramonakira**
This PR allows specifying frequency_lists in the ZXCVBNValidator options by adding a dict in the following form:

        {
            'NAME': 'pages.validators.CustomZXCVBNValidator',
            'OPTIONS': {
                'min_score': 3,
                'frequency_lists': {
                    'dutch': dutch_words,
                }
            }
        }

### CODE REVIEW

File: src/zxcvbn_password/validators.py [hunk 1]
```diff
@@ -20,7 +21,8 @@ class ZXCVBNValidator(object):
     DEFAULT_USER_ATTRIBUTES = ('username', 'first_name', 'last_name', 'email')
 
     def __init__(self, min_score=DEFAULT_MIN_SCORE,
-                 user_attributes=DEFAULT_USER_ATTRIBUTES):
+                 user_attributes=DEFAULT_USER_ATTRIBUTES,
+                 frequency_lists={}):
```
Comment (pawamoy) [inline src/zxcvbn_password/validators.py]: Prefer to use `frequency_lists=None` and a condition in the method body:

```python
if not frequency_lists:
    frequency_lists = {}
```

Using mutables as defaults in functions parameters is a code smell 🙂 (each time you instantiate ZXCVBNValidator, the same empty dict reference is used, so modifying it in instance A would also modify it in instance B).

Comment (ramonakira) [reply src/zxcvbn_password/validators.py]: Thanks! I changed it to

    self.frequency_lists = frequency_lists or {}

Hope, that's ok.

**Comment by pawamoy** [comment]
Perfect! Thanks 🙂 I'll try to release a new version ASAP.

**Comment by ramonakira** [comment]
Thanks so much!

**Comment by pawamoy** [comment]
I guess we could close #2? But the validation against custom dictionaries is only done server-side, right? We still don't have a way to do that client-side?

**Comment by ramonakira** [comment]
In my case the only thing that we do client-side is show the password strength bar. To make sure we get the same calculation in the bar as in the django's password validation we created a view that does the same call to zxcvbn as the password validation. Then we use that score as feedback in the password strength bar. 

This can be as simple as:

    class PasswordStrengthIndicatorView(View):

        def get(self, request, *args, **kwargs):
            add_frequency_lists({
                'dutch': dutch_words.get_ranked(),
            })

            password_strength = 0

            if request.GET.get('password'):
                password_strength = zxcvbn(request.GET.get('password'))['score']

            return HttpResponse(password_strength)

This has the advantage that the user does not need to download the (rather large) zxcvbn client-side library.

**Comment by pawamoy** [comment]
Hmmm, interesting. I guess this should have been done this way from the beginning. I'll open a new issue.

**Pull request merged**